### PR TITLE
Replace deprecated hibernate validator dependency into javax.validation

### PR DESCRIPTION
**Pull request by rajadilipkolli**
## Summary
Hibernate validator is deprecated, we need to use javax.validation hence making changes as per that.

### CODE REVIEW

File: azure-spring-boot/pom.xml [hunk 1]
```diff
@@ -24,12 +24,11 @@
...
-        <dependency>
-            <groupId>org.hibernate.validator</groupId>
-            <artifactId>hibernate-validator</artifactId>
-            <optional>true</optional>
-        </dependency>
-
+		<dependency>
+		    <groupId>javax.validation</groupId>
+		    <artifactId>validation-api</artifactId>
+		</dependency>
```
Comment (sophiaso) [inline azure-spring-boot/pom.xml]: Hi @rajadilipkolli, thank you for sending PR to fix deprecated dependencies. Could you change the indention to align with other dependencies?

Comment (rajadilipkolli) [reply azure-spring-boot/pom.xml]: @sophiaso , made changes to indention

Comment (rajadilipkolli) [reply azure-spring-boot/pom.xml]: @sophiaso , made changes to indention

Comment (sophiaso) [reply azure-spring-boot/pom.xml]: @rajadilipkolli  Thank you very much for the quick response.

**Pull request merged**